### PR TITLE
Create ShankarMadeshvaran

### PR DESCRIPTION
**Pull request by shankarmadeshvaran**
Add ShankarMadeshvaran as a Author

Blog Url: https://medium.com/a-developer-in-making/tagged/xamarin

Feed Url: https://medium.com/feed/a-developer-in-making/tagged/xamarin

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines (please check each item that applies):

- [x] I have a valid blog & RSS URL, both using HTTPS :lock: with a valid certificate
- [x] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [x] My blog is active with at least 3 Xamarin related blog posts in the last 6 months
- [x] In addition to the previous guideline; it should be apperant that the blog is active for a longer period of time and the posts on it are not fabricated in a small amount of time to comply to the '3 posts in last 6 months' guideline.
- [ ] I have applied a filter (Optional: if blog is only about Xamarin or you use `xamarin` in the title or categories in your blog posts)
- [x] I have added my Author class to the csproj file
- [x] I have implemented `IAmACommunityMember` or one of the other derivatives
- [x] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [x] I understand, my blog may be removed at any time if any of these guidelines are broken.
- [x] The submitted blog is not owned by a company which sole purpose it is to advertise a product of productline.

Only the collaborators of this repository decide whether a blog is eligible or not. They will determine if the rules are followed and if the quality of the content is high enough. 

Opening another PR, after the first one is closed in a small period of time will not be reevaluated. If a PR is closed, the issues with your blog are too significant to be fixed within a matter of hours.

### CODE REVIEW

File: src/Firehose.Web/Authors/ShankarMadeshvaran [hunk 1]
```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using System.Web;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+	public class ShankarMadeshvaran : IAmACommunityMember
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/ShankarMadeshvaran]: You need to add this in the csproj file as well.

Comment (Cheesebaron) [reply src/Firehose.Web/Authors/ShankarMadeshvaran]: File type needs to be `.cs`

Comment (shankarmadeshvaran) [reply src/Firehose.Web/Authors/ShankarMadeshvaran]: > csproj

where is csproj located

Comment (shankarmadeshvaran) [reply src/Firehose.Web/Authors/ShankarMadeshvaran]: how to added my author file to csproj

Comment (Cheesebaron) [reply src/Firehose.Web/Authors/ShankarMadeshvaran]: Clone your fork and open the solution in VS and add the file to the authors folder. Or simply edit the csproj file manually.

Comment (shankarmadeshvaran) [reply src/Firehose.Web/Authors/ShankarMadeshvaran]: i just edited csproj file manually via github editor and created another pull request.

Comment (Cheesebaron) [reply src/Firehose.Web/Authors/ShankarMadeshvaran]: Please stick to _one_ pull request.

Comment (shankarmadeshvaran) [reply src/Firehose.Web/Authors/ShankarMadeshvaran]: What should I do now? Did you edit something? or I need to add my author file to .csproj

Comment (shankarmadeshvaran) [reply src/Firehose.Web/Authors/ShankarMadeshvaran]: I think I need to change my blog Url and Feed Url. Should I delete this pull request and create a new one?

**Comment by shankarmadeshvaran** [comment]
I'm going to create a new pull request with diffrent blog urls and also with filter.Now I'm closing this pull request

**Pull request closed without merging**